src/main/telemetry/crsf.c

```diff
@@ -267,8 +267,17 @@ int16_t     Roll angle ( rad / 10000 )
 int16_t     Yaw angle ( rad / 10000 )
 */
 
-#define DECIDEGREES_TO_RADIANS10000(angle) ((int16_t)(1000.0f * (angle) * RAD))
+/// reduce angle to +/- 180 degree. Input: angle in 0.1 degree.
+static int16_t reduce_angle_range(int16_t angle_decidegree)
+{
+    while (angle_decidegree > 1800)
+        angle_decidegree -= 3600;
+    while (angle_decidegree < -1800)
+        angle_decidegree += 3600;
+    return angle_decidegree;
+}
 
+#define DECIDEGREES_TO_RADIANS10000(angle) ((int16_t)(1000.0f * reduce_angle_range(angle) * RAD))
```

Comment:
Missing newline after.

Also, using a define for this, but then having this define call a function is a bit of an antipattern - you can just as well change the entirety of this into a function.

Reply:
Agree. I just tried not to change much code.